module/Decision/src/Decision/Controller/AdminController.php

```diff
@@ -66,6 +66,42 @@ public function deleteDocumentAction()
         return $this->redirect()->toRoute('admin_decision/document');
     }
 
+    public function changePositionDocumentAction()
+    {
+        if (!$this->getRequest()->isPost()) {
+            // TODO: Throw 405 Method Not Allowed error
```

Comment:
Could someone provide guidance on how to create a new error page? Creating such a page might be benefical for other routes too.

Reply:
I think that what you want to throw is a 403. You can do so like this: 
```
throw new NotAllowedException(
    $translator->translate('You are not allowed to ...')
);
```